TimeOfFlight/README.md

```diff
@@ -1,174 +1,142 @@
-                                        . . .
-    parameter ParameterNames_MyTofClosest0ps [string]: momentumHM, trackLength, timeOfFlight,
-    parameter ParameterNames_MyTofFrankAvg50ps [string]: momentumHM, trackLength, timeOfFlight,
-    parameter ParameterNames_MyTofSET10ps [string]: momentumHM, trackLength, timeOfFlight,
-                                        . . .
+    parameter ParameterNames_MyTofClosest0ps [string]: timeOfFlight,
+    parameter ParameterNames_MyTofFrankAvg50ps [string]: timeOfFlight,
+    parameter ParameterNames_MyTofSET10ps [string]: timeOfFlight,
 
 
-Scrolling more down, one can see calculated parameters for each individual particle:
+... and see final results for each individual PFO
 
 
     ------------ detailed PID info: ---
       algorithms :                                        
-                                        . . .
-      [id: 9]   MyTofClosest0ps - params:  momentumHM trackLength timeOfFlight
-      [id: 11]   MyTofFrankAvg50ps - params:  momentumHM trackLength timeOfFlight
-      [id: 10]   MyTofSET10ps - params:  momentumHM trackLength timeOfFlight
-                                        . . .
+      [id: 9]   MyTofClosest0ps - params:  timeOfFlight
+      [id: 11]   MyTofFrankAvg50ps - params:  timeOfFlight
+      [id: 10]   MyTofSET10ps - params:  timeOfFlight
+
       [particle] |  PDG   | likelihood |  type  |  algoId  | parameters :
                  |        |            |        |          |              
       [00000073]                        . . .
-                 |      0 | 0.0000e+00 | 000000 |        9 | [ momentumHM : 3.84e+00, trackLength : 2.85e+03, timeOfFlight : 9.58e+00,]
-                 |      0 | 0.0000e+00 | 000000 |       10 | [ momentumHM : 3.84e+00, trackLength : 2.62e+03, timeOfFlight : 0.00e+00,]
-                 |      0 | 0.0000e+00 | 000000 |       11 | [ momentumHM : 3.84e+00, trackLength : 2.85e+03, timeOfFlight : 9.57e+00,]
-                                        . . .
+                 |      0 | 0.0000e+00 | 000000 |        9 | [ timeOfFlight : 9.58e+00,]
+                 |      0 | 0.0000e+00 | 000000 |       10 | [ timeOfFlight : 0.00e+00,]
+                 |      0 | 0.0000e+00 | 000000 |       11 | [ timeOfFlight : 9.57e+00,]
 
-## Extracting output parameters
+## Analysis
 
-Here is the pseudocode example on how one could extract PIDHandler parameters of *MyTofClosest0ps* results from the slcio file in your own Marlin processor and then use them to e.g. reconstruct the mass:
+After you run **TOFEstimators** and **TrackLength** processors you might want to run you analysis processor to e.g. calculate the mass of particles using time-of-flight information.
 
+Here is the code example how to do that:
 
-    void YourAmazingProcessor::processEvent(LCEvent* event){
+    float YourAmazingAnalysisProcessor::getParameterFromPID(ReconstructedParticle* pfo, PIDHandler& pidHandler, std::string algorithmName, std::string parameterName){
```

Comment:
You can make github highlight your code blocks in markdown via:
````
```cpp
// your code
```
````
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks